packages/lesswrong/components/form-components/ImageUpload.tsx

```diff
@@ -80,6 +80,13 @@ const cloudinaryArgsByImageType = {
     minImageWidth: 480,
     cropping: false,
     uploadPreset: cloudinaryUploadPresetEventImageSetting.get()
+  },
+  spotlightImageId: {
+    minImageHeight: 232,
+    minImageWidth: 345,
+    cropping: false,
+    // TODO: preset spotlight image url????
+    uploadPreset: cloudinaryUploadPresetGridImageSetting.get()
```

Comment:
Give this one its own db setting?  Not 100% sure what it's intended to be doing anyways